static/app.template.html

```diff
@@ -75,13 +76,13 @@
 aSummaryOptions = JSON.parse(JSON.stringify(options))
 aSummaryOptions.yaxis =  {
   tickFormatter: formatCurrency,
-  axisLabel: "Total Amount ({{.Currency}})",
```

Comment:
FIx currency here (removing the need for configuration)? Then we think about internationalization later.

With that, it will be easy to get app.js in the next PR :)